data/mships.txt

```diff
@@ -5152,9 +5152,9 @@ ship "AD-0032 Ialmuslati"
 	explode "small explosion" 30
 	explode "medium explosion" 20
 	"final explode" "final explosion medium"
-	description `Fast mid-sized transport offered by Najrmalas Shipworks seeing the few competitor. The Ialmuslati while expensive it offers more speed as well as business-class suite for better-paying customers.`
-	
-	#req|round10(bnk)|0
+	description `A fast mid-sized transport offered by Najrmalas Shipworks when they saw the few competitors in that field. The Ialmuslati, while expensive, offers more speed, as well as a business-class suite for better-paying customers.`
```

Comment:
```suggestion
	description `A fast mid-sized transport offered by Najrmalas Shipworks as they saw the few competitors in the market. The Ialmuslati, while expensive, offers more speed, as well as  a business-class suite for better-paying customers.`
```
Not 100% on the flow and phrasing but this should be better.

Reply:
That is better, but the space is definitely unneeded.
```suggestion
	description `A fast mid-sized transport offered by Najrmalas Shipworks as they saw the few competitors in the market. The Ialmuslati, while expensive, offers more speed, as well as a business-class suite for better-paying customers.`
```